packages/dds/tree/src/feature-libraries/flex-tree/lazyField.ts

```diff
@@ -132,6 +121,8 @@ export abstract class LazyField<TKind extends FlexFieldKind, TTypes extends Flex
 	}
 	public readonly key: FieldKey;
 
+	private readonly offAfterDestroy?: () => void;
```

Comment:
```suggestion
	/**
	 * If this field ends its lifetime before the Anchor does, this need to be removed to avoid a double free if the Anchor is destroyed.
	  */
	private readonly offAfterDestroy?: () => void;
```

Reply:
Thanks, tweaked the working/formatting and applied.